### PR TITLE
chore(frontend): Update push to GH action prompt

### PR DESCRIPTION
**Pull request by amanape**
**End-user friendly description of the problem this fixes or functionality that this introduces**

- [ ] Include this change in the Release Notes. If checked, you must provide an **end-user friendly** description for your change below

---
**Give a summary of what the PR does, explaining any non-trivial design decisions**
Updates the push to GitHub action prompt

---
**Link of any specific issues this addresses**

### CODE REVIEW

File: frontend/src/components/project-menu/ProjectMenuCard.tsx [hunk 1]
```diff
@@ -37,7 +37,8 @@ export function ProjectMenuCard({
 
   const handlePushToGitHub = () => {
     const rawEvent = {
-      content: "Please commit and push these changes to the repository.",
+      content:
+        "Please create a new branch and commit the changes. Then push them to the remote repository, and open up a pull request using the GitHub API and the token in the GH_TOKEN environment variable",
```
Comment (enyst) [inline frontend/src/components/project-menu/ProjectMenuCard.tsx:41]: Isn't this GITHUB_TOKEN?

Comment (amanape) [reply frontend/src/components/project-menu/ProjectMenuCard.tsx:41]: In the new UI, if we have the users GitHub token (e.g., after they sign in), we send a command event that exports the token in the shells environment so the agent can use it

https://github.com/All-Hands-AI/OpenHands/blob/e4590699417506b84c29970895a9be48a9f1fec8/frontend/src/routes/app.tsx#L125

Comment (enyst) [reply frontend/src/components/project-menu/ProjectMenuCard.tsx:41]: Sorry, I'm still confused. The var exists in a .ts file, in .md files including knowledge of a micro agent, and other places, and its name is GITHUB_TOKEN. I don't see how an agent will know about a GH_TOKEN instead. Why aren't we using GITHUB_TOKEN here too?

e.g. https://github.com/search?q=repo%3AAll-Hands-AI%2FOpenHands%20GITHUB_TOKEN&type=code

Comment (amanape) [reply frontend/src/components/project-menu/ProjectMenuCard.tsx]: Good catch, I see we even reference it as `GITHUB_TOKEN` in the user suggestions

https://github.com/All-Hands-AI/OpenHands/blob/59fbc11afeb4a7d9dd3ae4dd2a7e05b34421dd8b/frontend/src/utils/suggestions/non-repo-suggestions.ts#L9-L10

Comment (enyst) [reply frontend/src/components/project-menu/ProjectMenuCard.tsx]: Yes, exactly. Thanks for the fix!

**Review by rbren — APPROVED** [review]

**Pull request merged**